From e1484c0a91bfd2c6e8dde2aa85227c24eecd377b Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Fri, 20 Jul 2018 12:08:29 +0800
Subject: [PATCH 4463/5242] MLK-18987 soc: imx8: sc: types: Add SC_C_SYNC_CTRL
 into enum sc_ctrl_e

commit  a07637dd17ec24f1a1a0fad21bed1cd327b361cd from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds SC_C_SYNC_CTRL into enum sc_ctrl_e to sync with SCU
firmware commit <1db854d7d521> (SCF-151: Added new SC_C_SYNC_CTRL to
control both control signals at the same time.).

Signed-off-by: Liu Ying <victor.liu@nxp.com>
(cherry picked from commit 9e4a5892dad642f76e3bb0d46c77c4a59bbcae3d)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/soc/imx8/sc/types.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/soc/imx8/sc/types.h b/include/soc/imx8/sc/types.h
index 4ae035a..c2bb434 100644
--- a/include/soc/imx8/sc/types.h
+++ b/include/soc/imx8/sc/types.h
@@ -772,6 +772,7 @@
 	SC_C_IPG_STOP = 52,
 	SC_C_IPG_STOP_MODE = 53,
 	SC_C_IPG_STOP_ACK = 54,
+	SC_C_SYNC_CTRL = 55,
 	SC_C_LAST
 } sc_ctrl_t;
 
-- 
1.7.9.5

